src/layout/README.md

```diff
@@ -0,0 +1,72 @@
+# Layout Component
+
+## Usage
+
+* `Layout`: The layout wrapper, can contain `Header`, `Sider`, `Content`, `Footer` or `Layout` itself,
+* `Header`: The top layout, in which any element can be nested, and must be placed in `Layout`.
+* `Sider`: The sidebar with default style and basic functions, in which any element can be nested, and must be placed in `Layout`.
```

Comment:
can we name this `Sidebar`? i'd rather use a more common word

Reply:
that's a good point, done